jetstream/jetstream.go

```diff
@@ -197,6 +197,32 @@ type (
 		// DeleteConsumer removes a consumer with given name from a stream.
 		// If consumer does not exist, ErrConsumerNotFound is returned.
 		DeleteConsumer(ctx context.Context, stream string, consumer string) error
+
+		// CreateOrUpdateConsumer creates a push consumer on a given stream with
+		// given config. If consumer already exists, it will be updated (if
+		// possible). Consumer interface is returned, allowing to operate on a
+		// consumer (e.g. fetch messages).
```

Comment:
`e.g. fetch messages` is not a best example here  ;)

Reply:
Right, changed to "allowing to consume messages"